### PR TITLE
auto bundling & rpc methods

### PR DESCRIPTION
**Pull request by marie-fourier**
renamed relayer to executor
enabled auto bundling
rpc methods:
 - debug_bundler_setReputation
 - debug_bundler_dumpReputation
 - debug_bundler_setBundlingMode
 - debug_bundler_setBundleInterval

### CODE REVIEW

File: config.json.default [hunk 1]
```diff
@@ -10,7 +10,9 @@
       "rpcEndpoint": "http://127.0.0.1:8545",
```
Comment (ch4r10t33r) [inline config.json.default:10]: Can we set the default RPC port as 43371 (while the p2p port will be 43370)?

File: packages/executor/src/services/BundlingService.ts [hunk 2]
```diff
@@ -1,11 +1,15 @@
 import { BigNumber, ethers, providers } from "ethers";
 import { NetworkName } from "types/lib";
 import { EntryPoint__factory } from "types/lib/relayer/contracts/factories";
+import { EntryPoint } from "types/lib/relayer/contracts/EntryPoint";
+import { Mutex } from "async-mutex";
+import { SendBundleReturn } from "types/lib/relayer";
```
Comment (ch4r10t33r) [inline packages/executor/src/services/BundlingService.ts:6]: Shouldn't we rename `types/lib/relayer` to `types/lib/executor` ?

**Review by ch4r10t33r — CHANGES_REQUESTED** [review]

File: config.json.default (hunk 1 above)
Comment (marie-fourier) [reply config.json.default:10]: This is the endpoint for geth client - it's default port is 8545

File: packages/executor/src/services/BundlingService.ts (hunk 2 above)
Comment (marie-fourier) [reply packages/executor/src/services/BundlingService.ts]: renamed

**Review by ch4r10t33r — APPROVED** [review]

**Pull request merged**